app/main.py

```diff
@@ -1 +1,9 @@
-# write your code here
+def copy_file(command: str) -> None:
+    command_name, file_in, file_out, *_ = command.split()
+    if ("cp" in command_name
+            or file_in != file_out
+            or len(command_name) != 3):
```

Comment:
Rewrite the if condition
```suggestion
    if command_name == "cp" and file_in != file_out:
```
And after that place the logic under this if condition. There is no need to check for length here